transcribe.py

```
@@ -427,16 +473,28 @@ def transcribe_project(name, model, language, only=None):
             for seg in result["segments"]:
                 fh.write(f"[{fmt(seg['start'])} - {fmt(seg['end'])}] {seg['text'].strip()}\n")
-        dur = result["segments"][-1]["end"] if result["segments"] else 0
+        # `duration` ist die Laenge der AUFNAHME. Bis eben stand hier das Ende des letzten
+        # Segments unter der Beschriftung „Audio" — fehlen die letzten Fenster, meldet das
+        # genau die zu kurze Zahl, die den Verlust verdeckt. Der Rueckfall gilt Laeufen ohne
+        # das Feld (alte Roh-JSON, whisper.cpp ohne lesbare WAV).
+        dur = result.get("duration") or (result["segments"][-1]["end"] if result["segments"] else 0)
         print(f"[{name}] fertig {base}: {dt:.0f}s, {len(result['segments'])} Segmente, "
               f"Audio {fmt(dur)}, {dur/max(dt,1):.1f}x", flush=True)
+        if result["luecken"]:
+            orte = ", ".join(f"{fmt(l['start'])}-{fmt(l['end'])}" for l in result["luecken"])
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)